### DISCUSSION TITLE
Running Multiple "1 gNB 1 UE" scenario with Open5gs Core

### DISCUSSION THREAD

**Discussion by nishaadh** [srsran/srsRAN_Project#229] [General Help]
I am trying to run multiple gnBs connected to one UE each. I have different config files for all the UEs and gNBs but the problem I am facing right now is that the first gNB connects successfully and second one tries to bind to the same address and port number as the first one. So my question here is:

    1. Does the 5G Open5GS core allow multiple gNB connections?
    2. Is there a way to connect multiple gNBs on different ports to the same Open5gs core?

My console output for both gNBs:

    1. Successfully connected first gNB:
Lower PHY in executor blocking mode.

--== srsRAN gNB (commit 5e6f50a20) ==--

Connecting to AMF on 127.0.0.5:38412
Available radio types: zmq.
Cell pci=1, bw=10 MHz, dl_arfcn=368500 (n3), dl_freq=1842.5 MHz, dl_ssb_arfcn=368410, ul_freq=1747.5 MHz

==== gNodeB started ===

    3. Second gNB with error:
Lower PHY in executor blocking mode.

--== srsRAN gNB (commit 5e6f50a20) ==--

Connecting to AMF on 127.0.0.5:38412
Failed to bind UDP socket to 127.0.0.5:2152. Address already in use
Available radio types: zmq.
Cell pci=1, bw=10 MHz, dl_arfcn=368500 (n3), dl_freq=1842.5 MHz, dl_ssb_arfcn=368410, ul_freq=1747.5 MHz

==== gNodeB started ===

**Comment by pgawlowicz** [comment]
could you share your gnb configs?

**Comment by pgawlowicz** [comment]
Close as inactive
